test/sources.jl

```diff
@@ -20,12 +25,12 @@ using JSON
     @testset "Download and setup" begin
         mktempdir() do dir
             cd(dir) do
-                as = ArchiveSource("https://github.com/ralna/ARCHDefs/archive/v2.0.3x.tar.gz", "6583e27f84338447767bbdf4335514c8836ae4ad54f5e66280307e8b57189cff")
+                as = ArchiveSource("https://ftp.gnu.org/pub/gnu/auctex/auctex-11.81.tar.gz", "8e9e84515013cb79debbbc06f789817ca57c6125905e84f17a69339ae56632c1")
```

Comment:
I hope to find a ligher archive as source for this test, this is ~800 kiB, but the old one used to be ~16 kiB.